microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -223,6 +242,133 @@ def verify_kdumpcrash_large_memory_auto_size(
         self.is_auto = True
         self._kdump_test(node, log_path, log)
 
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. replace FRE bins with CHK bins and reboot the VM
```

Comment:
`Replace FRE bins with CHK bins and reboot the VM`
 If possible use full name for FRE and CHK?

Reply:
FRE / CHK is the term we use for hv builds. FRE is the release build and CHK is used for debug. Some of the things like hv crash will work with CHK build only. There is not full form as such. It is free/check build. I added this in description.